addons/account_peppol/models/account_move.py

```diff
@@ -70,3 +71,14 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = False
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    def _notify_by_email_prepare_rendering_context(self, message, **kwargs):
+        render_context = super()._notify_by_email_prepare_rendering_context(message, **kwargs)
+        invoice = render_context['record']
+        invoice_country = invoice.commercial_partner_id.country_code
+        if invoice_country in PEPPOL_DEFAULT_COUNTRIES:
+            render_context['peppol_info'] = {
+                'peppol_country': invoice_country,
+                'is_peppol_sent':  invoice.peppol_move_state in ('processing', 'done'),
```

Comment:
```suggestion
                'is_peppol_sent': invoice.peppol_move_state in ('processing', 'done'),
```

Reply:
<img src="https://media1.tenor.com/m/9uGXuqhhPpwAAAAC/the-penguins-of-madagascar-madagascar.gif" width="600" height="300" />